components/wso2is7.key.manager/src/main/java/org/wso2/is7/tenant/management/ISTenantManagementRestClient.java

```diff
@@ -0,0 +1,241 @@
+package org.wso2.is7.tenant.management;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * * REST Client for Managing Tenants in WSO2 Identity Server
+ */
+public class ISTenantManagementRestClient {
+    private static final String TENANT_ID = "ID";
+    private static final String TENANT_STATUS = "STATUS";
+    private static final String TENANT_OWNER_ID = "OWNER_ID";
+    private static final String TENANT_MANAGEMENT_RESOURCE_PATH = "api/server/v1/tenants/";
+
+    public static void createTenantInIS(String identityServerBaseUrl, String admin, String adminPassword,
+                                        String tenantDomain, String firstName,
+                                        String lastName, String email, String identityUserName,
+                                        String identityUserPassword) throws IOException {
+
+        String endpoint = identityServerBaseUrl + TENANT_MANAGEMENT_RESOURCE_PATH;
+
+        // JSON payload (no additional claims)
+        String payload = String.format("{%n" +
```

Comment:
Create a java class and use GSON to get a java to json object creation. do not build the json like this